index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Setup</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <div id="name"><a href="#">John Doe</a></div>
+            <nav class="toolbar">
+                <a id="hamburger" href="#"><img src="icons/menu.svg"></a>   
+            </nav>
+        </header>
+        <main>
+            <article class="headline-section">
+                    <h2>Hey There.<br>I’m Alex</h2>
+                    <h4>I’am a Software Developer</h4>
+                    <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.</p>
+                    <div class="socials">
+                        <a class="social" href=""><img src="icons/github.svg" /></a>
+                        <a class="social" href=""><img src="icons/linkedin.svg" /></a>
+                        <a class="social" href=""><img src="icons/angelist.svg" /></a>
+                        <a class="social" href=""><img src="icons/twitter.svg" /></a>
+                        <a class="social" href=""><img src="icons/medium.svg" /></a>
+                    </div>
+            </article>
```

Comment:
- [ ] This is the first section of your page and a `section` tag would fit better than an `article` tag. Kindly change that for better HTML semantics.